torch/testing/_internal/common_methods_invocations.py

```diff
@@ -192,6 +192,8 @@ class OpInfo(object):
     def __init__(self,
                  name,  # the string name of the function
                  *,
+                 ref=None,  # Just a reference
+                 sample_kwargs=lambda device, dtype, input: ({}, {}),
```

Comment:
Note: This won't be needed as for this case as we are iterating over sample inputs which has kwargs field. 
In case of Unary, the inputs are generated by tests so we need a way mechanism to pass other arguments.

Reply:
Ah, interesting! Thanks, will remove this.